Wrappers/Python/test/test_algorithms.py

```diff
@@ -569,12 +562,12 @@ def test_SPDHG_vs_PDHG_implicit(self):
             psnr(spdhg.get_output(), pdhg.get_output())
             )
         print ("Quality measures", qm)
-         
+            
         np.testing.assert_almost_equal( mae(spdhg.get_output(), pdhg.get_output()), 
-                                            0.0028578834608197212, decimal=3)
+                                            0.00033586719655431807, decimal=3)
```

Comment:
you're comparing to 3 decimals, so isn't this just  comapring it to 0.0, '0.00033586719655431807' seems excessive.

Reply:
done